### PR TITLE
Feature/upload function

### PR DESCRIPTION
**Pull request by Yairmendo**
Changes to load function in etl/main.py with new feature for update data base ( update to gasoline_station and gasoline_price tables allowing recurrent insertions) #22

### CODE REVIEW

File: etl/main.py [hunk 1]
```diff
@@ -238,9 +239,10 @@ def load(stations_dict):
         STATIONS_TABLE_NAME, metadata, autoload=True, autoload_with=engine)
 
     with engine.begin() as connection:
+        
         print('Inserting data...')
 
-        for record in stations_dict:
+        for record in stations_dict[:10]:
```
Comment (jcalvarezj) [inline etl/main.py]: Remove slicing

**Review by jcalvarezj — CHANGES_REQUESTED** [review]
Remove unnecessary slicing

**Comment by Yairmendo** [comment]
@gocastilloo  @AndresCendales  when you approve this PR please do a new PR to development

**Review by jcalvarezj — APPROVED** [review]

**Review by AndresCendales — CHANGES_REQUESTED** [review]
Please uncomment the requirements

**Pull request merged**